fuzz/cli.rs

```diff
@@ -312,9 +304,8 @@ fn run_libfuzzer(target: &str) -> Result<(), Error> {
          -C incremental=fuzz-incremental \
          -C passes=sancov \
          -C llvm-args=-sanitizer-coverage-level=4 \
-         -C llvm-args=-sanitizer-coverage-trace-pc-guard \
          -C llvm-args=-sanitizer-coverage-trace-compares \
-         -C llvm-args=-sanitizer-coverage-trace-divs \
+         -C llvm-args=-sanitizer-coverage-inline-8bit-counters \
```

Comment:
Why the change?

Reply:
>-fsanitize-coverage=trace-pc-guard is no longer supported by libFuzzer.
Please either migrate to a compiler that supports -fsanitize=fuzzer
or use an older version of libFuzzer

see [the ci script](https://github.com/rust-fuzz/libfuzzer/blob/master/ci/script.sh) of libfuzzer for details.